runway/hooks/staticsite/auth_at_edge/templates/http_headers/__init__.py

```diff
@@ -0,0 +1,23 @@
+
+# pylint: skip-file
+"""Add all configured headers (CloudFront compatable) to origin response.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
```

Comment:
At a glance it looks like this duplicate-code issue should be gone now.

Reply:
Still needed. It picks up on the redirect return values. Large discussion with Kyle about this in an earlier conversation. Leaving in-tact.

Reply:
Why can't the duplicate return lines be consolidated into a shared function?

Reply:
Upped the pylint requirement per your suggestion